MaxText/pyconfig.py

```diff
@@ -71,6 +67,12 @@ def validate_model_name(s: str) -> bool:
       "Invalid model name was passed. Valid options ", valid_model_names
     )
 
+def validate_overwritten_keys(keys1: list[str], keys2: list[str]):
+  overwritten_keys = [k for k in keys1 if k in keys2]
+  if overwritten_keys:
+    raise ValueError(
+            f"Keys {overwritten_keys} are overwritten from multiple sources. This isn't allowed.")
```

Comment:
Probably we could be clear that it is overwritten both from the model and from the env or command line?

Reply:
Sure, I will update that to be more specific. At the very beginning, I was thinking this would a helper func to validate any 2 lists of keys.